source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,228 @@
 ---
 title: HTML coding style
-last_reviewed_on: 2016-02-03
+last_reviewed_on: 2019-05-24
 review_in: 6 months
 ---
 
 # <%= current_page.data.title %>
 
-## HTML5
+## Progressive enhancement
 
-* Always quote attribute values - the decision tree for remember when to
-  selectively quote is too much mental overhead, so just quote them
-* Always use well-formed HTML - implicitly closed elements such as
-  `<p>` cause unnecessary parsing overhead and can cause issues in
-  accessibility
+HTML should be written in such a way that conforms with our [browser support model](https://github.com/alphagov/govuk-frontend#browser-support).
 
-## Use [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) roles appropriately
+For example, if IE8 is a supported browser, and it doesn't support the `<video>`
```

Comment:
``
```suggestion
For example, Internet Explorer 8 doesn't support the `<video>` element. If you need to support IE8, make sure that an appropriate fallback is in place:
```

Reply:
This suggestion runs over two lines as you've kept to a set number characters wide - which is good for code, but not so good for content 😃